sass/globals/_colors.sass

```diff
@@ -1,19 +1,19 @@
-$black: #000
-$white: #fff
+$black: #000 !default
+$white: #fff !default
```

Comment:
yeah you're probably right, I was adding default to everything to make it easily overwritable, this library's sass shouldn't be affecting projects, but it's probably safe to leave these without the default